source/mod_boinc.f90

```diff
@@ -0,0 +1,196 @@
+! ================================================================================================ !
+!  BOINC HELPER MODULE
+! ~~~~~~~~~~~~~~~~~~~~~
+!  V.K. Berglyd Olsen, BE-ABP-HSS
+!  Created: 2019-05-15
+!  Updated: 2019-05-15
+! ================================================================================================ !
+module mod_boinc
+
+  use floatPrecision
+
+  implicit none
+
+  integer,            private, save :: boinc_nTurn        = 0
+  integer,            private, save :: boinc_cpInterval   = -1
+  integer,            private, save :: boinc_progInterval = -1
+  integer,            private, save :: boinc_logUnit      = -1
+  character(len=12),  private, save :: boinc_logFile      = "cr_boinc.log"
+  character(len=256), private, save :: boinc_logBuffer    = " "
```

Comment:
It appends the turn number, and it's the same amount of calls anyway since it wraps the flush.
It also used to have the cpu time, but I removed that.